spec/result_spec.rb

```diff
@@ -45,4 +45,19 @@
       expect(result.coverage_f).to eq(1.0)
     end
   end
+
+  context 'for a branch without line coverage' do
+    let(:ast) { Imagen.from_local('spec/fixtures/module.rb') }
```

Comment:
I chose to use module.rb because branch related code is there but use `class.rb` doesn't really make a difference.